.gitignore

```diff
@@ -5,6 +5,7 @@ build/
 [Oo]bj/
 [Hh]elp/
 NuGetPackages/
+[pP]ackages
```

Comment:
I meant to annotate this in my PR as an inquiry - sorry for the oversight.  The nuget packages were showing up as needing to be added following my first compilation of the fork. What is the accepted way to handle this?